### PR TITLE
Update eluent introduction description & examples

### PR DESCRIPTION
**Pull request by mslarae13**
1- Update examples and description of eluent_introduction_category:

### CODE REVIEW

**Comment by mslarae13** [comment]
@turbomam see my commit https://github.com/microbiomedata/berkeley-schema-fy24/pull/169/commits/f508648067b39f6ade9c13aa0240e4c6227009ab

For line 1157-1160 , why is this commented out? Can it be deleted

**Comment by mslarae13** [comment]
> has_mass_spectrometry_configuration:
    range: MassSpectrometryConfiguration
    description: a description of how the mass spectrometry was configured

@kheal for this slot, in the example data you have ```has_mass_spectrometry_configuration: nmdc:mscon-99-oW43DzG0``` so, is it really a description? Or an identifier for the mass spec configuration file?

 > has_chromatography_configuration:
    range: ChromatographyConfiguration
    description: a description of how the chromatographic introduction was configured for the introduction of the sample into the mass spectrometer

I couldn't find an example for this one?

**Review by kheal — APPROVED** [review]
Whoops - to quick on the dial for the review to see you weren't finished :).

To answer your questions.  Yes you're correct re: `has_mass_spectrometry_configuration`.  It is an identifier for the associated `MassSpectrometryConfiguration` instance, so the description should be updated .  Similar logic for the `has_chromatogarphy_configuration`.  

There's an example of a `ChromatographyConfiguration` file here: https://github.com/microbiomedata/berkeley-schema-fy24/blob/main/src/data/valid/ChromatographyConfiguration-LC.yaml .  And an example of the `has_chromatogarphy_configuration` slot being used here: https://github.com/microbiomedata/berkeley-schema-fy24/blob/main/src/data/valid/Database-mass-spectrometry.yaml (this example also has an `ChromatographyConfiguration` example to show how they are connected).

**Comment by mslarae13** [comment]
Pending @turbomam 's response to https://github.com/microbiomedata/berkeley-schema-fy24/pull/169#issuecomment-2116368748

I am going to mark this PR ready for review since I did more than the title suggested & I don't want it to get too big..

File: src/schema/nmdc.yaml [hunk 1]
```diff
@@ -791,15 +791,18 @@ slots:
 
   eluent_introduction_category:
     range: EluentIntroductionCategoryEnum
-    description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
+    description: Details how a processed sample is introduced into a mass spectrometer
```
Comment (kheal) [inline src/schema/nmdc.yaml:794]: Can we modify this description to remove the word "Details"; this slot doesn't give details. 
 Maybe `A high-level description for how the processed sample is introduced into a mass spectrometer.`  Or something more elegant?

**Comment by mslarae13** [comment]
> @turbomam see my commit [f508648](https://github.com/microbiomedata/berkeley-schema-fy24/commit/f508648067b39f6ade9c13aa0240e4c6227009ab)
> 
> For line 1157-1160 , why is this commented out? Can it be deleted

From @turbomam  via slack "The commented definition of extractant  in https://github.com/microbiomedata/berkeley-schema-fy24/commit/f508648067b39f6ade9c13aa0240e4c6227009ab corresponds to it's dis-assocaiton form the Extraction class.
Extraction now uses the substances_used slot to link to any chemical that's used in the Extraction process, no matter it's role.
So you could delete the commented definition of extractant and the commented association with Extraction"

To do, delete these commented out lines

File: src/schema/nmdc.yaml (hunk 1 above)
Comment (mslarae13) [reply src/schema/nmdc.yaml]: changed in most recent commit

**Comment by mslarae13** [comment]
https://github.com/microbiomedata/issues/issues/705

File: src/schema/nmdc.yaml [hunk 2]
```diff
@@ -1146,7 +1149,11 @@ slots:
 
   contained_in:
     range: ContainerCategoryEnum
-    description: A type of container. i.e. test tube, falcon tube, etc.
```
Comment (turbomam) [inline src/schema/nmdc.yaml:1149]: Excellent! If use the examples structures to share examples.

File: src/schema/nmdc.yaml [hunk 3]
```diff
@@ -791,15 +791,18 @@ slots:
 
   eluent_introduction_category:
     range: EluentIntroductionCategoryEnum
-    description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
+    description: A high-level description for how the processed sample is introduced into a mass spectrometer.
```
Comment (turbomam) [inline src/schema/nmdc.yaml:794]: could this be a "high level categorization"?

File: src/schema/nmdc.yaml [hunk 4]
```diff
@@ -791,15 +791,18 @@ slots:
...
+    examples:
+      - value: liquid_chromatography
+      - value: direct_infusion_syringe
 
   has_mass_spectrometry_configuration:
     range: MassSpectrometryConfiguration
-    description: a description of how the mass spectrometry was configured
+    description: An identifier for the associated MassSpectrometryConfiguration.
```
Comment (turbomam) [inline src/schema/nmdc.yaml:801]: Yeah, "identifier" is better wording than "description". I recommend "A link to..." or "The identifier of..."

File: src/schema/nmdc.yaml [hunk 5]
```diff
@@ -791,15 +791,18 @@ slots:
...
 
   has_chromatography_configuration:
     range: ChromatographyConfiguration
-    description: a description of how the chromatographic introduction was configured for the introduction of the sample into the mass spectrometer
+    description: An identifier for the associated ChromatographyConfiguration, providing information about how a sample was introduced into the mass spectrometer.
```
Comment (turbomam) [inline src/schema/nmdc.yaml:805]: I suggest "A link to..." or "The identifier of..." as above.

File: src/schema/nmdc.yaml (hunk 3 above)
Comment (mslarae13) [reply src/schema/nmdc.yaml:794]: Yup!! I'll change that in the next PR

**Pull request merged**